sql/derived_tables/invoice_adjustments_ext.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE IF EXISTS folio_reporting.invoice_adjustments_ext;
+
+-- This table includes the percentage of the invoice lines amount in relation to the invoice_line total to calculate the invoice adjustment by row
+CREATE TABLE folio_reporting.invoice_adjustments_ext AS
+WITH invl_total AS (
+    SELECT
+        inv.id AS inv_id,
+        sum(invl.total) AS invl_total
+    FROM
+        INVOICE_INVOICES AS inv
+        LEFT JOIN invoice_lines AS invl ON inv.id = invl.invoice_id
+    GROUP BY
+        inv_id
+)
+SELECT
+    inv.id AS invoice_id,
+    invl.id AS invl_id,
+    CASE WHEN invl.total IS NULL THEN
+        0
+    ELSE
+        invl.total
+    END AS invoice_line_value,
+    CASE WHEN invadj.adjustment_value::decimal IS NULL THEN
+        0
+    ELSE
+        invadj.adjustment_value::decimal
+    END AS inv_adjust_total_value, --This is the total of the invoice adjustment 
+    invl.total AS "invl_total",
+    invltotal.invl_total AS "invls_total",
+    invadj.adjustment_prorate AS inv_adj_prorate,
+    invadj.adjustment_relationtototal AS inv_adj_relationToTotal,
+    invl.total::decimal / invltotal.invl_total::decimal AS perc_of_inv_adj_per_invoice_line, --This is the percentage of the invoice adjustment per invoice line
+    CASE WHEN invl.total IS NULL THEN
+        0
+    ELSE
+        invadj.adjustment_value::decimal * (
+            invl.total ::decimal / invltotal.invl_total)
+    END AS inv_adj_total
```

Comment:
It's a fraction. I'll make these changes.

Reply:
If it is a fraction rather than a percentage, should it called something different from `perc_...` which might imply a percentage?  Or it could be multiplied by 100 to get a percentage.

Also the number should be rounded I think.  For a fraction, rounding to 4 decimal places might be a good choice, e.g.: `round(..., 4)`.  For a percentage, maybe 1 decimal place?